app/views/items/_form.html.erb

```diff
@@ -1,7 +1,12 @@
 <%= simple_form_for @item, :html => {:multipart => true} do |f| %>
+
   <%= f.error_notification %>
 
   <div class="form-inputs">
+    <%= f.association :vendor, label: "Vendor ", collection: Vendor.order("name") %>
+
+    <%= link_to "Add a New Vendor", new_vendor_path(next: request.path) %>
```

Comment:
Could you please add a permissions check onto this ? Thanks

Reply:
👍